File: b/TimSepakbola.cpp
#include <iostream>
#include <string>

using namespace std;

class data{
	private:
		string tim;
		string negara;
		int tahun;
		int jumlah;
	
	public:
		data(){}
		
		data(string tim, string negara, int tahun, int jumlah){
			this->tim = tim;
			this->negara = negara;
			this->tahun = tahun;
			this->jumlah = jumlah;
		}
		
		void settim(string tim)
        {
            this->tim = tim;
        }
		string gettim(){
			return this->tim;
		}
		
		void setnegara(string negara){
			this->negara = negara;
		}
		string getnegara(){
			return this->negara;
		}
		
		void settahun(int tahun){
			this->tahun = tahun;
		}
		int gettahun(){
			return this->tahun;
		}
		
		void setjumlah(int jumlah){
			this->jumlah = jumlah;
		}
		int getjumlah(){
			return this->jumlah;
		}
		
		void output(){
			cout << "Nama Tim      : " << this->gettim() << endl;
			cout << "Negara Asal   : " << this->getnegara() << endl;
			cout << "Tahun Berdiri : " << this->gettahun() << endl;
			cout << "Jumlah Pemain : " << this->getjumlah() << endl;
		}
		
		~data(){
		}
};

File: b/TimSepakbola.php
<?php

class data
{
	//Deklarasi Variabel
	private $tim = "";
	private $negara = "";
	private $tahun = 0;
	private $jumlah = 0;
	
	
	//Construct
	public function __construct($tim = "", $negara = "", $tahun = 0, $jumlah = 0){
		$this->tim = $tim;
		$this->negara = $negara;
		$this->tahun = $tahun;
		$this->jumlah = $jumlah;
	}
	
	//Set dan get
	public function settim($tim){
		$this->tim = $tim;
	}
	public function gettim(){
		return $this->tim;
	}
	
	public function setnegara($negara){
		$this->negara = $negara;
	}
	public function getnegara(){
		return $this->negara;
	}
	
	public function settahun($tahun){
		$this->tahun = $tahun;
	}
	public function gettahun(){
		return $this->tahun;
	}
	
	public function setjumlah($jumlah){
		$this->jumlah = $jumlah;
	}
	public function getjumlah(){
		return $this->jumlah;
	}
	
	//output print
	public function output(){
	echo "Nama Tim      : ". $this->gettim(). "<br/>";
        echo "Asal Negara   : ". $this->getnegara(). "<br/>";
        echo "Tahun Berdiri : ". $this->gettahun(). "<br/>";
        echo "Jumlah Pemain : ". $this->getjumlah(). "<br/>";
	}
	
}
?>
